README.md

```diff
@@ -14,26 +15,20 @@ To add your project, please make a pull request. You'll need to create a data fi
 
 See [CONTRIBUTING.md](CONTRIBUTING.md).
 
-Approvers ([@open-telemetry/docs-approvers](https://github.com/orgs/open-telemetry/teams/docs-approvers))
+Roles: 
+- Approvers: [@open-telemetry/docs-approvers](https://github.com/orgs/open-telemetry/teams/docs-approvers)
+- Maintainers: [@open-telemetry/docs-maintainers](https://github.com/orgs/open-telemetry/teams/docs-maintainers))
+- Learn more about roles in the [community
+  repository](https://github.com/open-telemetry/community/blob/main/community-membership.md).
 
-- [Shelby Spees](https://github.com/shelbyspees), Honeycomb
-
-Maintainers ([@open-telemetry/docs-maintainers](https://github.com/orgs/open-telemetry/teams/docs-maintainers))
-
-- [Steve Flanders](https://github.com/flands), Splunk
-- [Morgan McLean](https://github.com/mtwo), Google
-- [Austin Parker](https://github.com/austinlparker), LightStep
-
-Learn more about roles in the [community repository](https://github.com/open-telemetry/community/blob/main/community-membership.md).
-
-Thanks to all the people who already contributed!
-
-<a href="https://github.com/open-telemetry/opentelemetry.io/graphs/contributors">
-  <img src="https://contributors-img.web.app/image?repo=open-telemetry/opentelemetry.io" />
-</a>
```

Comment:
I removed the image, IMHO it is visually overpowering w/o adding much value. I kept the link to the contributors tab.